<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>export_product</class>
 <widget class="QDialog" name="export_product">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Xuất kho</string>
  </property>
  <widget class="QPushButton" name="pb_export">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>300</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Xuất kho</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_product">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_cancel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>300</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Hủy</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_Tittle">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>Xuất kho</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_product">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Sản phẩm</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>221</width>
     <height>179</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Mã sản phẩm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_category">
        <property name="text">
         <string>Phân loại</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_supplier">
        <property name="text">
         <string>Nhà cung cấp</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_sl">
        <property name="text">
         <string>Số lượng hiện có</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_import_price">
        <property name="text">
         <string>Giá xuất</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="le_msp">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_category">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_supplier">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_so_luong_con">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_export_price">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>271</width>
     <height>119</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Đại lý</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Số lượng xuất</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Địa chỉ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Số điện thoại</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QComboBox" name="cb_dai_ly"/>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_so_luong_xuat">
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_dia_chiDL">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_sdtDL">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="le_maDL">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>160</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_maDL">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>110</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
